clueless/models.py

```diff
@@ -226,6 +226,7 @@ class Game(models.Model):
     startTime = models.DateTimeField(default = datetime.datetime.now, blank = True)
     lastUpdateTime = models.DateTimeField(default=datetime.datetime.now, blank=True)
     hostPlayer = models.ForeignKey(Player)
+    name = models.CharField(max_length=60)
 
```

Comment:
so when i run makemigrations i get an error about this "You are trying to add a non-nullable field 'name' to game without a default; we can't do that (the database needs something to populate existing rows)."

I'm guessing we need to do the same as with startTime and lastUpdateTime and add the default and blank. 

Edit: tested with changing that and it works, but I'll let you commit the change